allofplos/tests/test_unittests.py

```diff
@@ -211,6 +213,22 @@ def test_example_doi2(self):
         self.assertEqual(article.url[:100], "http://journals.plos.org/plosone/article/file?id=10.1371/annotation/3155a3e9-5fbe-435c-a07a-e9a4846e", 'url does not transform correctly for {}'.format(article.doi))
         self.assertEqual(article.word_count, 129, 'word_count does not transform correctly for {}'.format(article.doi))
 
+    def test_proofs(self):
+        """Tests whether uncorrected proofs and VOR updates are being detected correctly."""
+        article = Article(example_uncorrected_doi, directory=TESTDATADIR)
```

Comment:
you should be able to effect this using `os.environ['PLOS_CORPUS'] = os.path.abspath('test_corpus')` right? 
you shouldn't need to set the directory using `TESTDATADIR`…